src/hltas.cpp

```diff
@@ -614,34 +618,47 @@ namespace HLTAS
 						char *s2;
 						f.X = std::strtod(s, &s2); // TODO: replace this with lexical_cast probably.
 						f.Y = std::strtod(s2 + 1, nullptr);
+					} else if(str == "*") {
+						f.Yaw = lastYaw;
 					} else {
-						try {
-							f.Yaw = boost::lexical_cast<double>(str);
-						} catch (boost::bad_lexical_cast) {
-							throw ErrorCode::FAILFRAME;
+							try {
```

Comment:
You messed up the indentation on this block up until `break;`. It should be one level to the left.